src/tools/ceph_objectstore_tool.cc

```diff
@@ -2601,6 +2660,16 @@ int main(int argc, char **argv)
     goto out;
   }
 
+  if (object == "apply-layout-settings") {
```

Comment:
Yes, this is not object related so normally I prefer it not to be part of the positional arguments.   I think you can check the 'object' positional argument to hold the pool name.  Though the non-positional usage doesn't have a place to specify the --op needs the pool argument.

ceph-objectstore-tool ..... --op apply-layout-settings <pool>   

Alternatively, you could add an arg --apply-layout-settings which takes the pool.  There the usage is part of the add_options() at the start of main().

ceph-objectstore-tool ..... --apply-layout-settings <pool>